ParmedTools/ParmedActions.py

```diff
@@ -261,6 +265,8 @@ def execute(self):
         if self.rst_name is not None:
             if not Action.overwrite and os.path.exists(self.rst_name):
                 raise FileExists('%s exists; not overwriting.' % self.rst_name)
+        if self.vmd is not None:
+            self.parm.convert_to_vmd_compat()
```

Comment:
I don't think that `parmout` should modify the topology file -- it goes against what I would expect to happen.  So if you want the `vmd` option in `parmout` (and `outparm`), perhaps make a copy of `self.parm` and convert that to VMD compatibility... but this probably won't support `AmoebaParm` correctly yet...